portal/app/[locale]/_components/navbar/navbarDesktop.tsx

```diff
@@ -0,0 +1,110 @@
+'use client'
+
+import dynamic from 'next/dynamic'
+
+import { Badge } from '../badge'
+
+import { BitcoinKitLink } from './_components/bitcoinKitLink'
+import { BitcoinYield } from './_components/bitcoinYield'
+import { Dex } from './_components/dex'
+import { DocsLink } from './_components/docsLink'
+import { EcosystemLink } from './_components/ecosystemLink'
+import { GenesisDrop } from './_components/genesisDrop'
+import { GetStarted } from './_components/getStarted'
+import { HelpButton } from './_components/help/helpButton'
+import { HemiExplorerLink } from './_components/hemiExplorerLink'
+import { HemiStatusLink } from './_components/hemiStatusLink'
+import { HomeLink } from './_components/homeLink'
+import { NetworkSwitch } from './_components/networkSwitch'
+import { SocialLinks } from './_components/socialLinks'
+import { StakeDesktop } from './_components/stake'
+import { TunnelLink } from './_components/tunnelLink'
+import { Tvl } from './_components/tvl'
+import { VideoAsset } from './_components/videoAsset'
+
+const Separator = () => <div className="my-1 h-px w-full bg-neutral-100" />
+
+const Help = dynamic(() => import('./_components/help').then(mod => mod.Help), {
+  // Render the closed version of the help button
+  loading: () => <HelpButton isOpen={false} />,
+  ssr: false,
+})
+
+const PaddedListItem = ({
+  children,
+  className = '',
+}: {
+  children: React.ReactNode
```

Comment:
Here you also could import `ReactNode` on top of the file.

```suggestion
  children: ReactNode
```